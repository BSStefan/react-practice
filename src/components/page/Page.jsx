// @flow
import * as React from 'react';
import PageTitle from '../page-title/PageTitle';

type Props = {
  title: string,
  children?: React.Node
}

const Page = (props: Props) => {
  const { title, children } = props;
  return (
    <React.Fragment>
      <PageTitle title={title} />
      {children}
    </React.Fragment>
  )
};

export default Page;