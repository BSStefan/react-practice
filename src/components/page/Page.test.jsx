import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Page from './Page';

const defaultProps = {
  title: 'Test title',
  children: <div>Test</div>
}

it('Should match snapshout', () => {
  const tree = renderer.create(<Page {...defaultProps} />).toJSON();
  expect(tree).toMatchSnapshot();
})