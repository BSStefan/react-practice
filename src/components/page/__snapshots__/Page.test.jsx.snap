// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshout 1`] = `
Array [
  <h1
    className="sc-bdVaJa lnRlpM"
  >
    Test title
  </h1>,
  <div>
    Test
  </div>,
]
`;
